clients/src/test/java/org/apache/kafka/common/security/authenticator/SaslAuthenticatorTest.java

```diff
@@ -1471,8 +1473,9 @@ public void testCannotReauthenticateWithDifferentMechanism() throws Exception {
      * Second re-authentication must fail if it is sooner than one second after the first
      */
     @Test
-    public void testCannotReauthenticateAgainFasterThanOneSecond() throws Exception {
+    public void testCannotReauthenticateAgainFasterThanMaxReauth() throws Exception {
```

Comment:
@rajinisivaram I would keep this method name unchanged because the limitation really is a hard-coded 1 second as per https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/network/KafkaChannel.java#L530.  Other than that, though, LGTM.

Reply:
@rondagostino Thanks for the review. Sorry, I had forgotten that it was a hard-coded 1 second limit, reverted the name change.